src/types/DSNP.d.ts

```diff
@@ -1,132 +1,137 @@
 import { EthereumAddress } from "./Strings";
 
-export declare type DSNPMessage =
-  | DSNP.Broadcast
-  | DSNP.Reply
-  | DSNP.Drop
-  | DSNP.Reaction
-  | DSNP.GraphChange
-  | DSNP.KeyList
-  | DSNP.PrivateGraphKeyList
-  | DSNP.Private
-  | DSNP.EncryptionKeyList;
+export type ActionType = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
 
-export declare namespace DSNP {
-  /**
-   * Broadcast: a public post
-   */
-  export interface Broadcast {
-    /** fromAddress	ID of the sender	bytes20 */
-    fromAddress: EthereumAddress;
-    /** messageID	keccak-256 hash of content stored at URI	bytes32 */
-    messageID: Uint8Array;
-    /** uri	content URI	string */
-    uri: string;
-  }
+export const ActionTypePrivate = 0;
```

Comment:
I tried like mad to get namespaces to work and the advice I got from multiple sources was, give up on namespaces. So I do it here the "bonehead" way.